packages/apollo-server-fastify/README.md

```diff
@@ -12,6 +12,8 @@ npm install apollo-server-fastify graphql
 
 ## Fastify
 
+As of Apollo Server 3, this package works with Fastify v3.
```

Comment:
It _doesn't_ work with v2 anymore, correct? Do we think that's something people would also like to know here?

Reply:
I mean it's possible it does, I haven't tried it, but we certainly don't support that. Updating.